config.m4

```diff
@@ -342,6 +383,34 @@ if test "$PHP_MONGODB" != "no"; then
       PHP_MONGODB_ADD_BUILD_DIR([src/libmongoc/src/zlib-1.2.11/])
       AC_CONFIG_FILES([${ac_config_dir}/src/libmongoc/src/zlib-1.2.11/zconf.h])
     fi
+
+    if test "$PHP_CLIENT_SIDE_ENCRYPTION" = "yes"; then
+      dnl TODO: MONGOCRYPT-219 makes the -std argument obsolete
```

Comment:
Oh, this might be the comment I was expecting to appear above `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS`.

I was going to ask if we should just have `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS` defined here, but I see it's needed for libmongocrypt's `CheckSSL.m4`. Did we previously discuss moving the `PHP_MONGODB_ADD_SOURCES` out of that include and into this conditional?

If [MONGOCRYPT-218](https://jira.mongodb.org/browse/MONGOCRYPT-218) is now released, perhaps we don't even have to worry about this and can simply include _all_ source files all the time (as we do for libmongoc).